access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: Application Services
+:product: App Services
+:registry-product-long: OpenShift Service Registry
+:registry: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://console.redhat.com/beta/application-services/streams/
+:registry-url: https://console.redhat.com/beta/application-services/service-registry/
+:property-file-name: app-services.properties
+:rhoas-version: 0.29.0
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+:access-mgmt-url: access-mgmt/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-access"]
+= Managing account access in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: managing-access
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an owner of a Kafka instance in {product}, you can manage the level of access that other user accounts and service accounts have to your instance. You can allow or deny operations that specific accounts or that all accounts in your organization can perform for your instance or for certain resources in that instance, such as topics. You can also allow other users or service accounts to manage access to the resources in your instance for you.
+
```

Comment:
I'm tripping on the second sentence -- maybe the length or the order? I wish this wasn't an abstract, because a bulleted list would be awesome. 

Also, there's a bit of overlap between this abstract and the next one -- is there something that could be omitted from this one or vice versa?

Reply:
I know, it has really been something to try to explain things clearly but concisely. I can't do a ton about the overlap because the purpose of the abstract is to essentially give a digest of it, and so will inherently repeat key things, but I've tried to streamline a bit more.

So how about this, better?

>As an owner of a Kafka instance in {product}, you can manage the level of access that other user accounts and service accounts have to your instance. You can allow or deny access to your instance for specific accounts or for all accounts in your organization. You can also allow other users or service accounts to manage the level of access to your instance for you.

>You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.